File: lab1/hello_mpi.f90
program MPI

include "mpif.h"
integer myrank,size,ierr

call MPI_Init(ierr)
call MPI_Comm_rank(MPI_COMM_WORLD,myrank,ierr)
call MPI_Comm_size(MPI_COMM_WORLD,size,ierr)

write(*,*) "Processor ",myrank," of ",size,": Hello World!"

call MPI_Finalize(ierr)
end program

File: lab3/hello_mpi.c
#include <stdio.h>
#include <mpi.h>

int main (int argc, char *argv[])
{
  int myrank, size;

  MPI_Init(&argc, &argv);
  MPI_Comm_rank(MPI_COMM_WORLD, &myrank);
  MPI_Comm_size(MPI_COMM_WORLD, &size);
  printf("Processor %d of %d: Hello World!\n", myrank, size);

  MPI_Finalize();

  return 0;
}
